packages/example/src/pages/components/ExpressiveList.mdx

```diff
@@ -0,0 +1,135 @@
+  </ExpressiveList>
+    
+  <ExpressiveList 
+    title="Short Title"
+    titleType="expressive-02"
+    background="true">
+  Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate.
+  </ExpressiveList>
+</div>
+```
+
+<Title>With pictograms</Title>
+
+```mdx path=components/ExpressiveList/ExpressiveList.js src=https://github.com/carbon-design-system/gatsby-theme-carbon/tree/main/packages/gatsby-theme-carbon/src/components/ExpressiveList
+<div className="expressive-content-list-group">
+  <ExpressiveList 
+    title="Short Title"
+    background="true"
+    pictogram={<Connect className="my-custom-class" />}
+    >
+  Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate.
+  </ExpressiveList>
+
+  <ExpressiveList 
+    title="Short Title"
+    background="true"
+    pictogram={<Connect className="my-custom-class" />}
+    >
+  Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate.
+  </ExpressiveList>
+    
+  <ExpressiveList 
+    title="Short Title"
+    titleType="expressive-02"
+    background="true"
+    pictogram={<Connect className="my-custom-class" />}
+    >
+  Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate.
+  </ExpressiveList>
+</div>
+```
+
+### Props
+
+| property   | propType | required | default  | description                                                                                                        |
+| ---------- | -------- | -------- | -------- | ------------------------------------------------------------------------------------------------------------------ |
+| background | bool     |          | false    | Set true to add a background. Theme aware                                                                          |
+| children   | string   |          |          | Use to add the content. Takes in text content only                                                                 | 
+| className  | string   |          |          | Use to declare custom class name. This will be added on the outmost div                                            | 
+| picogram   | node     |          |          | Takes in a custom pictogram node from `carbon components` that is imported to the top of the page.                 |
+| title      | string   |          |          | Prop used for a short title                                                                                        |  
+| titleType  | string   |          |          | Optional prop used for type options - default is `expressive-04`. Options are: `expressive-03` and `expressive-02` |
```

Comment:
What is the default title type? Is there guidance on when to use which type? (probably a design @DianaStanciulescu question)

Reply:
I also wonder if it would be better to call it title size and have a small, medium and large? 🤔 @sstrubberg thoughts here? Do we have any text size props in any other components? (I don't think so, but not sure)

Reply:
> What is the default title type? Is there guidance on when to use which type? (probably a design @DianaStanciulescu question)

The title size depends on the amount of text and also on how the rest of the page looks like and where the list is inserted in the page layout. But I do not have exact specs for this, it;s left to each designer to choose. 

If we were to have a default, I think expressive heading 03 (medium) makes the most sense since it's the middle ground between the two.  I would like @jeanservaas 's opinion here too.

Reply:
I like `titleType` because the first thing that pops in my head is the [type tokens](https://carbon-elements.netlify.app/type/examples/preview/).